src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = {truck, bulldozer, excavator};
```

Comment:
It can be simplify to this for example:

```suggestion
        Machine[] machines = new Machine[]{new Bulldozer(), new Excavator(), new Truck()};
```